CODEOWNERS

```diff
@@ -1 +0,0 @@
- * @avrios/nebula
```

Comment:
removed as this repo is public/open source.

Reply:
@scornejob FYI